shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -42,7 +42,9 @@
   if (ios_surface_ || !owner_controller) {
     NotifyDestroyed();
     ios_surface_.reset();
-    accessibility_bridge_.reset();
+    if (owner_controller_) {
+      SetSemanticsEnabled(false);
```

Comment:
https://github.com/flutter/engine/blob/12e99e068ce6920bddb94b00e134936440fc5250/shell/platform/darwin/ios/platform_view_ios.mm#L118 makes me think this wouldn't work, which I find very confusing.  Ditto for below.

Reply:
How so?  I can get rid of it with refactoring.  It worked and kept the semantics of `PlatformViewIOS::SetSemanticsEnabled` so I didn't change it.

Reply:
The important thing is getting `PlatformView::SetSemanticsEnabled(false);` called because that percolates the need to clear things out on the Dart side.

Reply:
Oh, nevermind, this is `if (owner_controller_)`, not `if (!owner_controller_)`.  This makes sense now.